esmvalcore/cmor/_fixes/cmip6/awi_esm_1_1_lr.py

```diff
@@ -21,9 +21,9 @@ def fix_metadata(self, cubes):
         bad_value = 'days since 0000-01-01 00:00:00'
         for cube in cubes:
             try:
-                if cube.attributes[parent_units] == bad_value:
-                    cube.attributes[parent_units] = 'days since 0001-01-01 ' \
-                        + '00:00:00'
+                if parent_units in cube.attributes and \
+                    cube.attributes[parent_units] == bad_value:
+                        cube.attributes[parent_units] = 'days since 0001-01-01 00:00:00'
```

Comment:
```suggestion
                if parent_units in cube.attributes:
                    if cube.attributes[parent_units] == bad_value:
                        cube.attributes[parent_units] = \
                            'days since 0001-01-01 00:00:00'
```